modules/augment/src/flip.cpp

```diff
@@ -23,5 +23,26 @@ namespace augment {
         Mat imageM = image.getMat();
         return Point2d(imageM.size().width - 1 - src.x, src.y);
     }
+
+
+
+
+
+    FlipVertical::FlipVertical(const Scalar& _probability) : Transform(_probability) {}
+
+
+    void FlipVertical::image(InputArray _src, OutputArray _dst)
+    {
+        Mat src = _src.getMat();
+        _dst.create(src.size(), src.type());
+        Mat dst = _dst.getMat();
+        cv::flip(src, dst, 0);
```

Comment:
Have you tried to use just `cv::flip(_src, _dst, 0);`?

Reply:
Ok will try it